src/components/NewClusterModal.js

```diff
@@ -236,7 +236,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
         }
       }, [
         div({ style: { fontSize: '0.875rem', fontWeight: 600, marginBottom: '0.5rem' } }, ['COMPUTE POWER']),
-        div({ style: { marginBottom: '1rem' } }, ['Select from one of the compute runtime profiles or define your own']),
+        div({ style: { marginBottom: '1rem' } }, ['Select from one of the compute instance profiles or define your own']),
```

Comment:
I believe this text change is covered by another ticket but in the mock this text is: "Select from one of the default compute cluster profiles or define your own"